packages/warehouses/.eslintrc.json

```diff
@@ -0,0 +1,16 @@
+{
+  "extends": [
+    "eslint:recommended",
+    "airbnb-base",
+    "airbnb-typescript/base",
+    "prettier",
+    "plugin:json/recommended"
+  ],
+  "plugins": ["@typescript-eslint"],
+  "parserOptions": {
+    "project": "./tsconfig.json"
+  },
+  "rules": {
+    "no-console": "off"
```

Comment:
you probably don't need this rule in this package

Reply:
thanks!